packages/neuron-wallet/jest.config.js

```diff
@@ -18,5 +18,10 @@ module.exports = {
     "json",
     "node"
   ],
-  setupFiles: ['<rootDir>/tests/setup.ts']
+  setupFiles: ['<rootDir>/tests/setup.ts'],
```

Comment:
We currently have the `tests/setup.ts` to setup the global mocks. It would be better to put all the global mocks in the same place.

Reply:
The `queue.test.ts` fails all the time and `queue.intg.test.ts` fails sometimes when the `electron` is mocked globally.
All most all tests fail when `fs` is mocked globally.
Should I fix them all?

Reply:
Ok, I see. Sounds like if `electron` is mocked globally, then there could be conflicts with the mocks in some of the tests.

If this is the case, then I would suggest the mocks for `electron` shouldn't be controlled within the individual tests instead of globally.